### DISCUSSION TITLE
WyzeCam V3 Pro Support

### DISCUSSION THREAD

**Discussion by SomebodySysop** [gtxaspec/wz_mini_hacks#537] [Development]
After getting beat down here: https://github.com/gtxaspec/wz_mini_hacks/issues/536

I am wondering:  What would it take to get wz_mini_hacks support for the new V3 Pro camera?

**Comment by endertable** [comment]
It would take someone to first hack the V3 Pro (get root) so that stuff could be altered.  wz_mini_hacks works off the fact that Wyze left some back door testing facilities open on the boot FW on prior cam models and/or allowed FW archives to be altered without checks.  These have all been "fixed" or obfuscated on the V3 Pro and beyond.

So until someone gets wyzer and finds another way in, we are stuck with using the facilities as set forth by Wyze.  But depending on what your needs are, there are some other nice hacks out there that work with the cams out of the box, per se.  The following is very popular:

https://github.com/mrlt8/docker-wyze-bridge
